utils/validation/user.ts

```diff
@@ -0,0 +1,108 @@
+import { locale, roles, province } from ".prisma/client";
+import { UserInput, ParentInput, VolunteerInput } from "models/User";
+import validator from "validator";
+
+/**
+ * Valdaites whether a given province has been specified correctly
+ * @param userProvince - the province to validate
+ * @returns - true if the province is valid, false otherwise
+ */
+function validateProvince(userProvince) {
+    return [
+        province.AB,
```

Comment:
Not sure if this would be better but perhaps we can make this a const outside the function

Reply:
Also, would it be better to use the Set object if we want to check for existence